core/src/main/java/nl/nn/adapterframework/core/PipeLine.java

```diff
@@ -49,15 +49,12 @@
 import nl.nn.adapterframework.util.Misc;
 
 /**
- * Processor and keeper of a line of {@link IPipe Pipes}.
- * <br/>
- * Pipelines also generate statics information per Pipe and keep forwards, that are registered
- * at individual pipes during the configure phase.
- * <br/>
- * In the AppConstants there may be a property named "log.logIntermediaryResults" (true/false)
- * which indicates whether the intermediary results (between calling pipes) have to be logged.
+ * Required in each {@link Adapter} to transform incoming messages. A pipeline
+ * is a network of pipes that are connected using {@link PipeForward}-s. A
```

Comment:
Ik zou zeggen: een sequence of pipes. PipeForwards komen later wel.

Reply:
Gedaan. Ik heb een verwijzing naar PipeForward toegevoegd. Ik weet dat global forwards in deze PR nog niet goed worden uitgelegd. Dat wou ik graag uitstellen tot een volgende PR.